frontend/app/js/script.js

```diff
@@ -29,11 +29,18 @@ $(function() {
     $("a[data-action='enlarge']",codeDiv).show();
   });
 
-  $('form input[type=submit], form button[type=submit]').on('click', function() {
-    var $this = $(this);
+  $('form').on('submit', function() {
+    var $this = $(this).find('button');
```

Comment:
It seems what the function was doing was finding the button that's related to the form and disabling it on click. With form on submit, it will find the related button and disable much in the same way, but with the purpose of disabling it if you click on it or send the form with ⌘+Enter.